SVProgressHUD/UIImage+Framework.h

```diff
@@ -0,0 +1,15 @@
+//
+//  UIImage+Framework.h
+//  SVProgressHUD
+//
+//  Created by Loïs Di Qual on 2/16/15.
+//  Copyright (c) 2015 EmbeddedSources. All rights reserved.
+//
+
+#import <UIKit/UIKit.h>
+
+@interface UIImage (Framework)
+
++ (UIImage *)imageNamed:(NSString *)name classInFramework:(Class)aClass;
```

Comment:
Agreed. What do you propose? A regular function? A method inside `SVProgressHUD.m`? Keeping in mind that it'll need to be accessible in `SVIndefiniteAnimatedView.m` too.